Neuro/Recording/RecordingHandler.cs

```diff
@@ -57,19 +57,20 @@ public void FixedUpdate()
 
         // string frameString = JsonSerializer.Serialize(frame);
         // Info(frameString);
+        ResetFrameData();
+    }
+
+    private void ResetFrameData()
+    {
 
         DidReport = DidVent = DidKill = DidSabotage = DidDoors = false;
-        SabotageUsed = SabotageTypes.None;
+        // TODO: Figure out a default value for this or make nullable!
+        SabotageUsed = SystemTypes.Hallway;
```

Comment:
```suggestion
        SabotageUsed = default;
```
Even though `default` is `Hallway`, in this case we don't care about that, we just want to have a value representing null or no sabotage used.

Reply:
Yep, changed in my next push.